### PR TITLE
Diona clothes Mk.2

### PR DESCRIPTION
**Pull request by SkyCinnamon**
## About the Pull Request

Pr is an improved version of #61 , Diona have every clothes slot except shoes and gloves as per suggestions. hud glasses are limited to only eyepatches for diona and other things were addressed which wouldnt make sense. diona can also now wear hard hats with lights (Light given from them isnt enough to prevent death anyways)

## Why It's Good For The Game

More clothe slots are good, this also helps diona with jobs in minor ways while not disrupting balance.

## Did you test it?

Hell yeah brother!

## Changelog

🆑 
tweak: diona now have mask , left ear, eye slots
tweak: diona can now wear hard hats
rscadd: diona cant wear hud glasses but can wear eyepatch huds
rscadd: you cannot blindfold a diona 
/:cl:

### CODE REVIEW

File: code/modules/clothing/glasses/thermals.dm [hunk 1]
```diff
@@ -38,10 +39,11 @@
 	icon_state = "eyepatch"
 	item_state = "eyepatch"
 	body_parts_covered = 0
+	species_restricted = null
```
Comment (Shadowtail117) [inline code/modules/clothing/glasses/thermals.dm:42]: Any reason you have this?

Comment (SkyCinnamon) [reply code/modules/clothing/glasses/thermals.dm:42]: its to allow diona to wear them and bypass the restriction of all the thermal subtypes

Comment (Shadowtail117) [reply code/modules/clothing/glasses/thermals.dm:42]: Ah gotcha. The collapsed rest of the file made it difficult to see in context. Seems good to me.

**Review by Cupax3 — APPROVED** [review]
LGTM

**Pull request merged**